<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankMainWindow</class>
 <widget class="QMainWindow" name="BankMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bank Management System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="createTab">
       <attribute name="title">
        <string>Create Account</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Full Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nameInput"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Account Type:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="typeCombo">
          <item>
           <property name="text">
            <string>Savings Account</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Current Account</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Initial Deposit:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="depositSpin">
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1000000.000000000000000</double>
          </property>
          <property name="prefix">
           <string>$</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="createBtn">
          <property name="text">
           <string>Create Account</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="createStatus">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="customerTab">
       <attribute name="title">
        <string>Customer Login</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Login</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Account ID:</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="custIdInput"/>
           </item>
           <item>
            <widget class="QPushButton" name="loginBtn">
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="custOperations">
          <property name="title">
           <string>Account Operations</string>
          </property>
          <property name="enabled">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0" colspan="3">
            <widget class="QLabel" name="custInfo">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="depositBtn">
             <property name="text">
              <string>Deposit</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="withdrawBtn">
             <property name="text">
              <string>Withdraw</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="transferBtn">
             <property name="text">
              <string>Transfer</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="3">
            <widget class="QListWidget" name="transactionsList">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>150</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="logoutBtn">
             <property name="text">
              <string>Logout</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="adminTab">
       <attribute name="title">
        <string>Admin Panel</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Admin Login</string>
          </property>
          <layout class="QFormLayout" name="formLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Password:</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="adminPassInput">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="adminLoginBtn">
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="adminOperations">
          <property name="title">
           <string>Admin Operations</string>
          </property>
          <property name="enabled">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0" rowspan="4">
            <widget class="QListWidget" name="accountsList">
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="blockBtn">
             <property name="text">
              <string>Block Account</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="unblockBtn">
             <property name="text">
              <string>Unblock Account</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="changeNameBtn">
             <property name="text">
              <string>Change Name</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Search:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="searchInput"/>
             </item>
             <item>
              <widget class="QPushButton" name="searchBtn">
               <property name="text">
                <string>Search</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QLabel" name="adminStatus">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>